### PR TITLE
Added Trigger Bot option to KillAura

### PR DESCRIPTION
**Pull request by lasnikprogram**
You can be dumb and copy all the killaura code to then implement Triggerbot

<h2>or</h2>

You can be smart **(like me - lasnik)** and make an ArrayList with one entry to implement Triggerbot

### CODE REVIEW

File: BleachHack-Fabric-1.16/src/main/java/bleach/hack/module/mods/Killaura.java [hunk 1]
```diff
@@ -64,31 +67,35 @@ public void onTick(EventTick event) {
...
+						|| (e instanceof ArmorStandEntity && getSetting(4).asToggle().state))
 				.sorted((a, b) -> Float.compare(a.distanceTo(mc.player), b.distanceTo(mc.player)))
 				.findFirst();
 
 		if (target.isPresent()) {
 			Entity e = target.get();
 			
-			if (getSetting(4).asRotate().state) {
-				WorldUtils.facePosAuto(e.getX(), e.getY() + e.getHeight() / 2, e.getZ(), getSetting(4).asRotate());
+			if (getSetting(5).asRotate().state && getSetting(0).asToggle().state) {
```
Comment (BleachDrinker420) [inline BleachHack-Fabric-1.16/src/main/java/bleach/hack/module/mods/Killaura.java:93]:
```suggestion
			if (getSetting(5).asRotate().state && !getSetting(0).asToggle().state) {

```

Comment (lasnikprogram) [reply BleachHack-Fabric-1.16/src/main/java/bleach/hack/module/mods/Killaura.java:93]: Yea. Sorry I was speedrunning

**Comment by BleachDrinker420** [comment]
AAAAAAA
i really want to hate on that arrayListWithOneEntry code but i literally can't think of a better way of doing it

**Comment by lasnikprogram** [comment]
You can´t hate on beatiful code...

**Comment by lasnikprogram** [comment]
I updated the code... Maybe this is cleaner. I have no idea

**Comment by BleachDrinker420** [comment]
Manually added

**Pull request closed without merging**